CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# global owners as fallback for anything not matched below
+* @KlausHaeuptleSAP @HrFlorianHoffmann @xtough
+
+# one line for each document
+/abap-code-review/ @xtough
```

Comment:
Could there be a second owner from the ABAP team? In case you are on vacation.

Reply:
the file itself is just suggestions, its part of the branch protection rules if reviews are required, and if reviews from code owners are required

Reply:
Exactly. The idea is not to force reviews from codeowners (I hope at least), because this might be detrimental to fostering open source contributions.